### DISCUSSION TITLE
Plugin presets not available

### DISCUSSION THREAD

**Discussion by Henri75** [apohl79/audiogridder#1204] [Help]
I noticed that for a lot of plugins, VST3 presets (with right click on the plugin name in AG) are not available.

For example with Plugin Alliance plugins

**Comment by apohl79** [comment]
Will have to investigate. Thx.

**Reply by Loupap** [reply]
Has this issue been solved yet? I am not able to find or use any of the user made presets for a VST3 plugin such as trackspacer. Is there a directory path that needs to be considered when using Audiogridder?

**Reply by apohl79** [reply]
I think if you create custom preset within a plugin, the plugin is not reporting it to the host as "builtin" preset. Thats why AG does not see it. You could create an AG preset instead.

**Comment by Henri75** [comment]
Hi,

A lot of plugins are not showing their factory presets too.

By the way, I made a request a long time ago for AG beeing able to not
reload all the chain when using AG internal presets on the same plugin
chain.
Currently it is a pain : loading AG presets can take dozens of seconds
because everytime it reload the chain, whereas it should be only updating
each plugin settings if the plugin chain is the same.
Same issue if you use your DAW presets : chain is always reloading even if
it is the same plugin(s)

That would be a game changer to use AG.

Plus a previous/next preset selector would be useful to browse preset :)

I remember too that you were thinking about drag/and drop between AG
instances, that would be awesome



Le mer. 18 oct. 2023 à 10:38, Andreas Pohl ***@***.***> a
écrit :
